docs/react/bootstrap.md

```diff
@@ -44,6 +44,7 @@ It will create a folder called `my-app` inside the current folder with the follo
 ```console
 my-app
 ├── node_modules
+├── .gitignore
```

Comment:
Fix sort order.

```suggestion
├── .gitignore
├── node_modules
```

Reply:
This was the order i got from when using ```tree``` command

Reply:
That is... strange. What command do you use?

Try the following to go 2 levels deep, show all files including invisibles, and ignore the `node_modules` directory.

```
tree -L 2 -a -I node_modules
```

Reply:
In my output, `.` files are first, then files and directories that start with an uppercase letter, then files and directories that start with a lowercase letter.

Reply:
It was mistake that .gitignore was before the node_modules....i have updates the mistake...pls review it